app/stuquiz/entities/course.py

```diff
@@ -16,7 +16,7 @@ class Course(Entity):
     university_id: str
     name: str
     description: str
-    professor: str
+    professor_id: str
```

Comment:
Maybe we can use something else as professor_id instead of string, in order to have unique id for each professor